#input_clusters_topic: "laser_clustering/clusters"
input_clusters_topic: "laser_overlap_trace/clusters"
results4meters_topic: "~results4meters"
frame_id: "laser_link"
#classifier_file: "SVM_classifier.p" #place the file inside the classification_files folder.
#classifier_file: "LDA_classifier.p" #place the file inside the classification_files folder.
classifier_file: "GaussianNB_classifier.p"
publish_viz: True
viz_topic: "~viz_req"
dt: 25
human_speed: 5
distance: 4
min_distance: 1
timewindow: 40
write_to_file: False
file: '/home/rosturtle/Desktop/hprStat.csv'
pca_file: "PCA_object.p"  #place the file inside the classification_files folder.
